From 916188f8b7ba4b501f92fd26a6a6aa77fe45d2ca Mon Sep 17 00:00:00 2001
From: furrymcgee <furrymcgee@lippydanger.jumpingcrab.com>
Date: Thu, 15 Aug 2019 12:29:54 +0200
Subject: [PATCH 4/4] Can't exec "dh_strip_nondeterminism": No such file or
 directory

Disable as they are unneeded (and we can then be sure debhelper builds without needing autotools-dev, dh-strip-nondetermism etc.)
---
 debian/rules | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index e78e574..2f2aa92 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,7 +1,8 @@
 #!/usr/bin/make -f
 
 %:
-	DEB_RULES_REQUIRES_ROOT=no dh $@ --without autoreconf
+	DEB_RULES_REQUIRES_ROOT=no dh $@ \
+		--without autoreconf
 
 # Remove the auto-generated Makefile to prevent dh_auto_clean from failing
 # if the Makefile contain paths referring to an old version of Perl.
-- 
2.23.0.rc1

